### DISCUSSION TITLE
Getting negative values from getElevationForLngLatZoom

### DISCUSSION THREAD

**Discussion by jean-kiepura** [maplibre/maplibre-gl-js#4409] [Q&A] [answered]
Hello,

I have a raster-dem source that's not very far from sea level, and some areas of that DEM is under sea level (something like -12 meters).

When I"m querying the dem with `map.terrain.getElevationForLngLatZoom()`, I get zero instead of the expected negative value.

Is this a known limitation? 

The following image show the console output for a point where we're supposed to have a -12 meters elevation

![image](https://github.com/user-attachments/assets/4d04abcc-d4dd-4b1c-8b2e-b29d445b46ca)

**Comment by HarelM** [comment, marked as answer]
I don't think there's a known limitation, but there are some issues with elevation relative to screen center. 
See:
https://github.com/maplibre/maplibre-gl-js/pull/3854

**Reply by jean-kiepura** [reply]
Yes indeed, it was a code 18, something was wrong in our terrain-RGB generation, the following image demonstrates elevations below zero.

Thanks for the answer!

![image](https://github.com/user-attachments/assets/8aec500c-7100-4e91-90f9-a53596d39ea6)
